api.go

```diff
@@ -1,6 +1,16 @@
 package stacker
 
+import "fmt"
+
 const (
-	GitVersionAnnotation      = "com.cisco.stacker.git_version"
-	StackerContentsAnnotation = "com.cisco.stacker.stacker_yaml"
+	GitVersionAnnotation      = "com.%s.stacker.git_version"
```

Comment:
remove the "com." as well

For example, this would "io.zotregistry.stacker.git_version"

Reply:
Thinking about this some more. Do we want to just default this to "io.project-stacker.stacker.git_version"

Reply:
"io.stackeroci.stacker.git_version"

Reply:
ok, sure